config/config.go

```diff
@@ -73,18 +95,12 @@ func (config *Config) Validate(path string) ([]string, error) {
 		return nil, utils.NewConfigValidationFieldRequiredError(path, "config_params[mode]")
 	}
 
-	if config.CloudStoryEnabled {
-		if config.ExistingMap == "" {
-			return nil, utils.NewConfigValidationFieldRequiredError(path, "existing_map")
-		}
-	} else {
-		if config.DataDirectory == "" {
-			return nil, utils.NewConfigValidationFieldRequiredError(path, "data_dir")
-		}
+	if config.DataDirectory == "" {
+		return nil, utils.NewConfigValidationFieldRequiredError(path, "data_dir")
+	}
 
-		if config.MapRateSec != nil && *config.MapRateSec < 0 {
-			return nil, errors.New("cannot specify map_rate_sec less than zero")
-		}
+	if config.MapRateSec != nil && *config.MapRateSec < 0 {
```

Comment:
Looks like we are still checking for MapRateSec here - since it is not a part of the camera dictionary, I think the above lines (71-80) can just be removed. Unless there is something you are trying to do as part of the PR that I am missing

Reply:
you are right - sorry about the confusion